sdk/eventgrid/azure-eventgrid/tests/test_eg_publisher_client_async.py

```diff
@@ -287,4 +287,4 @@ async def test_send_and_close_async_session(self, resource_group, eventgrid_topi
                     data = "cloudevent",
                     type="Sample.Cloud.Event"
                     )
-            await client.send_events(cloud_event)
\ No newline at end of file
+            await client.send_events(cloud_event)
```

Comment:
I think you meant this?
```suggestion
            await client.send(cloud_event)
```